Extra tire

ocorella opened the pull request.

ocorella committed:
Add extra tire select, fix margins and add select title

ocorella committed:
Protractor tests

ocorella committed:
Karma tests

ocorella committed:
Karma tests

ocorella committed:
Added tireBrand constants

ocorella committed:
Pull from develop

ocorella committed:
Integrate constants with tireRevision view

ocorella committed:
Integrate lastRevision service with tireRevision

joserv2812 commented on tests/e2e/carInfo/carInfo.spec.js:
```diff
@@ -0,0 +1,8 @@
+
+  it('should go to carInfo page', function() {
+    browser.get(browser.baseUrl + 'carInfo');
+    expect(browser.getCurrentUrl()).toEqual("http://localhost:8100/#/carInfo");
```
use browser.baseUrl in toEqual("http://localhost:8100/#/carInfo");

joserv2812 commented on tests/e2e/carView/carView.spec.js:
```diff
@@ -0,0 +1,15 @@
+
+  it('should go to carView page', function() {
+    browser.get(browser.baseUrl + 'carView');
+    expect(browser.getCurrentUrl()).toEqual("http://localhost:8100/#/carView");
```
use browser.baseUrl in toEqual

joserv2812 commented on tests/e2e/login/login.spec.js:
```diff
@@ -1,38 +1,31 @@
   beforeEach(function() {
-  	  browser.get('');	
-  	  browser.sleep(500); 
+      browser.get('');  
```
fix indentation

joserv2812 commented on tests/e2e/tireRevision/tireRevision.spec.js:
```diff
@@ -0,0 +1,18 @@
+
+  it('should go to tireRevision page', function() {
+    browser.get(browser.baseUrl + 'tireRevision');
+    expect(browser.getCurrentUrl()).toEqual("http://localhost:8100/#/tireRevision");
```
use browser.baseUrl in toEqual

joserv2812 commented on tests/specs/feedback/feedback.ctrl.spec.js:
```diff
@@ -0,0 +1,20 @@
+/*describe('Feedback Controller', function() { 
```
why this file is commented?

ocorella replied:
Johnny is going to make some changes in the controller, so I will wait to mock it properly

joserv2812 commented on tests/specs/login/login.services.spec.js:
```diff
@@ -0,0 +1,37 @@
+    it('is defined', function() {
+        expect(LoginFirebaseService).toBeDefined();
+      });
+    /*it('fills new revision data ', function() {
```
you can use xit to ignore the case

ocorella committed:
fix pr
Conflicts:
	www/app/tireRevision/tireRevision.controller.js
	www/app/tireRevision/tireRevision.service.js

joserv2812 commented on tests/e2e/carDeliveryInfo/carDeliveryInfo.spec.js:
```diff
@@ -1,20 +1,11 @@
-    browser.get('http://localhost:8100/#/carDeliveryInfo');
-    expect(browser.getCurrentUrl()).toEqual("http://localhost:8100/#/carDeliveryInfo")
+    browser.get(browser.baseUrl + 'carDeliveryInfo');
+    expect(browser.getCurrentUrl()).toEqual("http://localhost:8100/#/carDeliveryInfo");
```
use browser.baseUrl

joserv2812 commented:
resolve conflicts

joserv2812 commented:
squash commits

ocorella committed:
Merge branch 'develop' of https://github.com/pernixgit/Budget-RentaCar into ExtraTire

Conflicts:
	www/app/tireRevision/tireRevision.controller.js
	www/app/tireRevision/tireRevision.html

ocorella committed:
indent
